egs2/bibletts/tts1/run.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+fs=22050
+n_fft=1024
+n_shift=256
+
+opts=
+if [ "${fs}" -eq 22050 ]; then
+    # To suppress recreation, specify wav format
+    opts="--audio_format wav "
+else
+    opts="--audio_format flac "
+fi
+
+train_set=tr_no_dev
+valid_set=dev
+test_sets="dev eval1"
+
+train_config=conf/train.yaml
+inference_config=conf/decode.yaml
+
+# g2p=g2p_en # Include word separator
+g2p=g2p_en_no_space # Include no word separator
```

Comment:
Why do you use an English g2p for modeling? I thought this recipe was for multilingual purposes.

Your results seem to be with char? please align with your model with the setting~

Reply:
I set g2p=none and token_type=char in the running command in "README.md" file. I will also modify them in the "run.sh" file